src/native/Cargo.toml

```diff
@@ -12,9 +12,11 @@ opt-level = 3
 pyo3 = { version = "0.24", features = ["extension-module"] }
 datadog-ddsketch = { git = "https://github.com/DataDog/libdatadog", rev = "v16.0.3" }
 datadog-library-config = { git = "https://github.com/DataDog/libdatadog", rev = "v16.0.3" }
+ddcommon = { git = "https://github.com/DataDog/libdatadog", rev = "99056cf717cfe9ee066ab853b5a562a3026c8ced" }
```

Comment:
Why is it necessary to depend on a separate copy of libdatadog, and is it possible to use one of the existing dependencies without adding another?

Reply:
That's a valid concern @emmettbutler. These are two independent crates from the same `libdatadog` repo. Unless we introduce an unnecessary dependency between them, I am afraid it is not possible to reuse the `datadog-library-config` dependency for service discovery.

Reply:
@emmettbutler do you have any clues on why `Build cpXX-manylinux_i686` fails?

Reply:
This error message is deep in the job's log output.
```
error[E0308]: mismatched types
          --> /root/.cargo/git/checkouts/libdatadog-fb268e227e330049/99056cf/ddcommon/src/rate_limiter.rs:57:22
           |
        57 |         (ts.tv_sec * TIME_PER_SECOND + ts.tv_nsec) as u64
           |                      ^^^^^^^^^^^^^^^ expected `i32`, found `i64`
        
        error[E0277]: cannot multiply `i32` by `i64`
``` @dmehala